docs/development_suite/identity-and-access-management/group-management.md

```diff
@@ -0,0 +1,107 @@
+:::
+
+<!-- <div style={{display: 'flex', justifyContent: 'center'}}>
+  <div style={{display: 'flex', width: '600px'}}>
+
+TODO: ![Add group](./img/group-management/add_user.png)
+
+  </div>
+</div> -->
+
+It is possible to add to a Group both Users who already have access to the Company and new Users who do not have access to the Company (but which already exist on the Platform). For the latter, once inserted their email address, they will receive an invitation to join that Group of the Company.  As long as they are part of at least one Group of a Company, they can have access to the resources of that Company.
+
+The provided emails will be searched throughout existing users on the Platform and, if a match is found, the user will be invited to the Group with the defined Role of that specific Group.
+
+:::info
+If the User does not exist in the Platform, please open a Service Request to invite it to register in the Platform.
+:::
+
+### Editing a Group
+
+After the creation of a Group, it is possible to enter its detail page.
+Once entered, a Company Owner can perform the following actions:
+
+* Change Group name
+* Edit Group Company role
+* Add new members to the Group
+* Remove members from the Group
+
+:::info
+In the case that a User has access to the Company solely from its membership of that Group, the removal from that Group will result in loss of access to the Company for that User.  
+
+In the case that a User has access to more Company Groups, or has a specific individual Company role assigned, the removal from that Group will NOT result in the loss of access to the Company for that User; for this reason, it is necessary to pay close attention to what privileges are still left to that User on the Company resources. 
+:::
+
+<!-- 
+<div style={{display: 'flex', justifyContent: 'center'}}>
+  <div style={{display: 'flex', width: '600px'}}>
+
+TODO: ![Edit group](./img/group-management/edit_group.png)
+
+  </div>
+</div> -->
+
+### Deleting a Group
+
+A Group can be deleted by accessing its detail page and clicking on the *Delete* button at the end of the page and then confirming the action.
+
+:::caution
+Group deletion may result in loss of access to the Company for those Group members who are not members of at least one other Company Group or who do not have a specific individual Company role assigned.   
+
+In the case that a Group member has access to the Company solely from its membership of the deleted Group, the removal from that Group will result in the loss of access to the Company for that User.  
+
+In the case that a User has access to more Company Groups, or has a specific individual Company role assigned, the deletion of the Group will NOT result in loss of access to the Company for that User; for this reason, it is necessary to pay attention to what privileges are still left to that User on the Company resources.
```

Comment:
This concept is repeated here and for group member removal

Reply:
I thought it was useful to reiterate because both actions have an impact on that topic, so if a person were to read only one of the two paragraphs, they would find that specific information